scalameta/semanticdb-scalac-core/src/main/scala/scala/meta/internal/semanticdb/DocumentOps.scala

```diff
@@ -153,7 +153,21 @@ trait DocumentOps { self: DatabaseOps =>
               // https://github.com/scalameta/scalameta/issues/665
               // Instead of crashing with "unsupported file", we ignore these cases.
               if (gsym0 == null) return
-              if (gsym0.isAnonymousClass) return
+              if (gsym0.isAnonymousClass) {
+                if (config.members.isAll) {
+                  gsym0.tpe.parents
+                    .map(_.typeSymbol)
+                    .filterNot(_ == g.definitions.ObjectClass)
+                    .foreach { parent =>
+                      val symbol = parent.toSemantic
+                      if (!members.contains(symbol)) {
```

Comment:
Do we have a map for handled scala symbols? Would be good to guard there before `.toSemantic`

Reply:
Is this optimization necessary? Is `gsym0.asClass.parentSymbols` expensive to compute?